package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "1.0.4-beta",
+	"version": "1.1.0-beta",
```

```suggestion
	"version": "1.1.0",
```

I think we can remove beta from the version name.

We are stable enough to just be `1.1.0`